docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -241,19 +241,44 @@ In order to use a Nessie catalog, ensure to configure the catalog type with
 properties:
 
 .. list-table:: Nessie catalog configuration properties
-  :widths: 40, 60
+  :widths: 30, 58, 12
   :header-rows: 1
 
   * - Property name
     - Description
+    - Default
   * - ``iceberg.nessie-catalog.uri``
     - Nessie API endpoint URI (required).
       Example: ``https://localhost:19120/api/v1``
+    -
   * - ``iceberg.nessie-catalog.ref``
-    - The branch/tag to use for Nessie, defaults to ``main``.
+    - The branch/tag to use for Nessie.
+    - ``main``
   * - ``iceberg.nessie-catalog.default-warehouse-dir``
     - Default warehouse directory for schemas created without an explicit
       ``location`` property. Example: ``/tmp``
+    -
+  * - ``iceberg.nessie-catalog.transport.read-timeout``
+    - The read timeout duration for requests to the Nessie server.
```

Comment:
Link to duration type and inline default value with

```
Defaults to ``25s``.
```

Reply:
> Link to duration type

Duration is commonly used in may existing configuration and none of the `.rst` files links it. So, this hyperlinking is not necessary?

Reply:
@mosabua: So, handled all the comments except hyperlinking. Please check. Thanks for the review.

Reply:
ok. I have added the hyper link now. 
Since I couldn't find hosted java docs for airlift classes. I hyperlinked to the code.

Reply:
Updated the hyperlink as per the suggestion to 
:ref:`duraction <prop-type-duration>`